jiant/shared/model_resolution.py

```diff
@@ -218,3 +218,10 @@ def resolve_tokenizer_class(model_type):
 
     """
     return TOKENIZER_CLASS_DICT[ModelArchitectures.from_model_type(model_type)]
+
+
+def resolve_is_lower_case(tokenizer):
+    if isinstance(tokenizer, (transformers.BertTokenizer, transformers.AlbertTokenizer)):
```

Comment:
not future-proof ie: adding ELECTRA. Would prefer something like the following:
`if hasattr(a, 'property'):
    a.property`

Reply:
Let's add this as an issue and follow up in a new PR.

Reply:
https://github.com/jiant-dev/jiant/issues/41